conda/_vendor/tqdm/utils.py

```diff
@@ -365,7 +365,7 @@ def _screen_shape_linux(fp):  # pragma: no cover
             return cols, rows
         except:
             try:
-                return [int(os.environ[i]) - 1 for i in ("COLUMNS", "LINES")]
+                return [int(os.environ.get(i, 1)) - 1 for i in ("COLUMNS", "LINES")]
```

Comment:
If either of the environment variables are not set (return empty string), `int()` will fail.

Reply:
This would still fail is `$COLUMNS` or `$LINES` were set to an empty string (e.g., `COLUMNS=`) or some other non-integer value. Perhaps a better approach would be to change the `except KeyError:` below to `except KeyError, ValueError`?

Reply:
Good point!
I will fix the PR.

Reply:
Thanks!